src/main/java/nextstep/courses/domain/Session.java

```diff
@@ -0,0 +1,119 @@
+package nextstep.courses.domain;
+
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Session {
+    private Long id;
+    private String image;
```

Comment:
```suggestion
    private final Long id;
    private final String image;
```

인스턴스 변수는 가능하면 final로 구현을 시작하고 변경이 필요한 시점에 final을 제거하는 방식으로 구현하면 어떨까?